lib/postPrepareHook.js

```diff
@@ -52,8 +53,11 @@ module.exports = function(context) {
             return cspRemover.Remove();
         });
     }).fail(function(err) {
-        var msg = pluginName + ': An error occurred' + JSON.stringify(err);
-        events.emit('warn', err);
-        return Q.reject(msg);
+        var msg = pluginName + ' - An error occurred: ' + err;
+        //events.emit('warn', msg);
```

Comment:
It ends up not doing anything. It doesn't display the message, so it's misleading to leave it there.